### ISSUE TITLE
userInput_ variables are not detected when used in expressions

### ISSUE THREAD

**Issue by rjeczalik** [koding/koding#11208]
For the following stack template, we correctly detect and ask for `Test` variable:

```
provider:
  aws:
    access_key: '${var.aws_access_key}'
    secret_key: '${var.aws_secret_key}'
resource:
  aws_instance:
    aws-instance:
      instance_type: 't2.nano'
      tags:
        Name: '${var.koding_user_username}-${var.koding_group_slug}'
      user_data: |-
        echo "${var.userInput_test}!" >> /helloworld.txt
```

However we do not detect test variable in the following template:

```
provider:
  aws:
    access_key: '${var.aws_access_key}'
    secret_key: '${var.aws_secret_key}'
resource:
  aws_instance:
    aws-instance:
      instance_type: 't2.nano'
      tags:
        Name: '${var.koding_user_username}-${var.koding_group_slug}'
      user_data: |-
        echo "${base64encode(var.userInput_test)}!" >> /helloworld.txt
```

**Comment by gokmen** [comment]
@rjeczalik right, regex for that one won't detect https://github.com/koding/koding/blob/master/client/app/lib/util/stacks/requirementsparser.coffee#L52 I'll take a look at it.

**Comment by gokmen** [comment]
fixed with #11211
